.travis.yml

```diff
@@ -18,11 +18,9 @@ install:
   - pip install 'matplotlib==1.5.2' tox
   - pip install .
 script:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then tox -e py27; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then tox -e py27 -e coverage; fi
   - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then tox -e py35; fi
   - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then tox -e py36 -e lint; fi
```

Comment:
should we be doing the coverage on a newer python?

Reply:
Good point. I changed that.